### ISSUE TITLE
Train multi-label problem

### ISSUE THREAD

**Issue by ommiissyu** [dmlc/mxnet#1658]
hi, 

I am trying to train a multi-label model. I found that symbol.SoftmaxOutput support multi-output. For single output, the SoftmaxOutput's input usually is Fullyconnected layer. But for muti-output, what kind of layer should I use? Can anyone give a example of using softmaxoutput's multi-output?

**Comment by piiswrong** [comment]
Reshape your fullyconnected layers output to 3 dimensions. (batch_size, num_output, num_class)

**Comment by ommiissyu** [comment]
Thanks @piiswrong.

**Comment by EasonD3** [comment]
@leoxiaobin, can you please note down your final solution here? Thanks.

**Comment by Strateus** [comment]
@piiswrong are you sure in the sequence?
I have 10 outputs and every output is integer from 0 to 6.
I made it (batch_size, 7, 10) and only then it worked, otherwise shape was inconsistent.

**Comment by miraclewkf** [comment]
There is a example of multi-label image classification: https://github.com/miraclewkf/multilabel-MXNet @piiswrong @leoxiaobin @EasonD3 @Strateus
